### PR TITLE
[Agent] add license config

### PR DESCRIPTION
**Pull request by TomatoMr**
### This PR is for:

- Agent
- Server

#### Checklist
- [ ] Added unit test.
#### Backport to branches
- main

### CODE REVIEW

File: server/controller/model/agent_group_config_example.yaml [hunk 1]
```diff
@@ -1285,6 +1285,18 @@ vtap_group_id: g-xxxxxx
   ##   https://unix.stackexchange.com/questions/495854/processes-hanging-when-trying-to-access-a-file
   #check-core-file-disabled: false
 
+  ############################
+  ## Authorization function ##
+  ############################
+  ## When it is false, it supports the integration of java-async-profiler, golang-pprof and other pile insertion analysis data
+  #external-profile-integration-disabled: false
```
Comment (sharang) [inline server/controller/model/agent_group_config_example.yaml:1292]: 格式统一一下

Comment (sharang) [reply server/controller/model/agent_group_config_example.yaml:1292]: image

Comment (sharang) [reply server/controller/model/agent_group_config_example.yaml:1292]: and other pile insertion analysis data =>

and other profile data by instrumentation.

File: server/controller/model/agent_group_config_example.yaml [hunk 2]
```diff
@@ -1285,6 +1285,18 @@ vtap_group_id: g-xxxxxx
   ##   https://unix.stackexchange.com/questions/495854/processes-hanging-when-trying-to-access-a-file
   #check-core-file-disabled: false
 
+  ############################
+  ## Authorization function ##
```
Comment (sharang) [inline server/controller/model/agent_group_config_example.yaml:1289]: External Signals

File: server/controller/model/agent_group_config_example.yaml (hunk 1 above)
Comment (TomatoMr) [reply server/controller/model/agent_group_config_example.yaml:1293]: ok

**Review by sharang — APPROVED** [review]

**Review by sharang — APPROVED** [review]

**Pull request merged**